### DISCUSSION TITLE
What is the release strategy for the Alpha version?

### DISCUSSION THREAD

**Discussion by cary-rowen** [nvaccess/nvda#16267] [General]
Hello NV Access,

I have a question about the release strategy for the NVDA alpha version that I'm hoping to get some clarity on.
As I understand it, the release process seems automated: a new PR is merged into the master branch, followed by a brief waiting period for the CI build to complete, and then the Alpha version is released. Is this an accurate depiction of the workflow?
However, I've noticed instances where the Alpha version doesn't seem to be released as promptly as expected. For example, today, several PRs were merged, but an update to the Alpha version hasn't been observed. Could you provide some insight into this? Are there specific reasons for the delay, or is there a different release schedule that I might not be aware of?

Thank you for your time and dedication to the development and maintenance of NVDA.
Warm regards,
Cary

**Comment by Brian1Gaff** [comment]
Well, I was under the impression, maybe wrongly, that this had to be forced as a choice at the time. Local testing might  end up with  an issue that could be fixed before it goes public and produces a snapshot.
 Brian

-- 
***@***.*** 
Sent via blueyonder.(Virgin media) 
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
  ----- Original Message ----- 
  From: Rowen 
  To: nvaccess/nvda 
  Cc: Subscribed 
  Sent: Wednesday, March 06, 2024 9:28 AM
  Subject: [nvaccess/nvda] What is the release strategy for the Alpha version? (Discussion #16267)



  Hello NV Access,

  I have a question about the release strategy for the NVDA alpha version that I'm hoping to get some clarity on.
  As I understand it, the release process seems automated: a new PR is merged into the master branch, followed by a brief waiting period for the CI build to complete, and then the Alpha version is released. Is this an accurate depiction of the workflow?
  However, I've noticed instances where the Alpha version doesn't seem to be released as promptly as expected. For example, today, several PRs were merged, but an update to the Alpha version hasn't been observed. Could you provide some insight into this? Are there specific reasons for the delay, or is there a different release schedule that I might not be aware of?

  Thank you for your time and dedication to the development and maintenance of NVDA.
  Warm regards,
  Cary

**Comment by lukaszgo1** [comment]
Yes, normally Alpha builds are released automatically. However after PR #16071 got merged our CI fails as NVDA cannot be installed on a machine without any sound card. As a result no new Alpha build can be compiled.

**Comment by Adriani90** [comment]
@lukaszgo1 that's interesting, that means it was not part of the system tests right?
cc: @gerald-hartig, @michaelDCurran

**Reply by lukaszgo1** [reply]
See #16269  - all our system tests were failing on that PR, but it was merged regardless.
